metricbeat/module/system/process/helper.go

```diff
@@ -216,7 +221,7 @@ func getProcState(b byte) string {
 	return "unknown"
 }
 
-func (procStats *ProcStats) GetProcessEvent(process *Process, last *Process) common.MapStr {
+func (procStats *ProcStats) GetProcessEvent(process *Process) common.MapStr {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method ProcStats.GetProcessEvent should have comment or be unexported